src/components/Preview.jsx

```diff
@@ -13,7 +13,7 @@ class Preview extends React.Component {
     bindAll(this, '_handlePopOutClick');
   }
 
-  _generateDocument() {
+  _generateDocument(isPopout = false) {
```

Comment:
@alexpelan right, sorry, that was confusing. my first comment was just wrong. second comment is basically saying that the preexisting code is *also* wrong—we should be varying the values of *all* the options passed to `_generatePreview` from the `_generateDocument` method. However I am down to address that separately since it’s outside the scope of this change. Making a ticket now!